javascript/ql/test/library-tests/frameworks/SQL/sqlite.js

```diff
@@ -1,10 +1,34 @@
-// Adapted from https://github.com/mapbox/node-sqlite3/wiki/API, which is
-// part of the node-sqlite3 project, which is licensed under the BSD 3-Clause
-// License; see file node-sqlite3-LICENSE.
-var sqlite = require('sqlite3');
+import sqlite3 from 'sqlite3'
+import { open } from 'sqlite'
+
+const unsafe = "unsafe"
```

Comment:
What you're testing here is whether values are recognized as SQL strings. 
You are not testing SQL-injection, so having this "unsafe" string doesn't really make sense.  

You could improve how you test by instead adding to the tests for SQL-injection.  
That way the you could use an actually unsafe value in your tests.